addons/account/populate/account_bank_statement.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from odoo import models
+from odoo.tools import populate
+from datetime import datetime, timedelta
+from functools import lru_cache
+
+_logger = logging.getLogger(__name__)
+
+class AccountBankStatement(models.Model):
+    _inherit = "account.bank.statement"
+    _populate_sizes = {
+        'small': 10,
+        'medium': 10,
+        'large': 1000,
+    }
+
+    _populate_dependencies = ['account.journal']
+
+    def _populate_factories(self):
+        @lru_cache()
+        def search_journal_ids(company_id):
+            return self.env['account.journal'].search([('company_id', '=', company_id), ('type', 'in', ('cash', 'bank'))]).ids
```

Comment:
Actually, I want to go for the existing journals too because I chose to install a CoA on the companies, but I want to add a few journals on top of it.

I use a real CoA because I don't see why we would want to test with different sizes since all the companies are going to have something very similar, regardless of the size of it.

Reply:
I would suggest you ton only use xmlids and populated_models. Else, adding a new data/demo will change the populated records, and we are trying to make something as much stable as possible.